index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <p>Alejandro Ramos</p>
+        <img src="images/IconMenu.png" alt="Three horizontal dashes button">
+    </header>
```

Comment:
- [ ] Great job using the `<header>` as the container for the header section, however, there are some issues we will need to fix 😉 . the name `Alejandro Ramos` is considered as the logo, which means it has to be something on which we can click, like the button or a link, but the `<p>` tag you used can not help us achieve this, so it could be good to use the `<a>` tag instead of the `<p>` tag, this is going to keep our code more semantic 👍 


- [ ] We can improve the header section by doing the same thing for the hamburger icon as well since we actually need to click on the hamburger to show the hidden menu links in the mobile view, so having a `<button>` or `<a>` tag as the container of the hamburger image(`<img src="images/IconMenu.png" alt="Three horizontal dashes button">`) makes more sense. After this, we will need to have a `<nav>` tag as the container of the hamburger button, since the hamburger represents the hidden links that are supposed to be located in the `<nav>` tag, Kindly let's fix this 😉

Reply:
Thanks a lot for your suggestion 👍